app/src/features/refusjon-omsorgspenger-arbeidsgiver/Steg2AnsattOgArbeidsgiver.tsx

```diff
@@ -27,26 +46,61 @@ export const RefusjonOmsorgspengerArbeidsgiverSteg2 = () => {
           <TextField
             className="flex-1"
             label="Ansattes fødselsnummer (11 siffer)"
+            onChange={(e) => {
+              setFødselsnummer(e.target.value);
+            }}
           />
           <div className="flex-1">
             <Label>Navn</Label>
-            <BodyShort>Navnerud Orama</BodyShort>
-          </div>
-        </div>
-      </Informasjonsseksjon>
-      <Informasjonsseksjon kilde="Fra Altinn" tittel="Arbeidsgiver">
-        {/* TODO: Legg til støtte for flere arbeidsforhold */}
-        <div className="flex gap-4">
-          <div className="flex-1">
-            <Label>Virksomhetsnavn</Label>
-            <BodyShort>Proff Arbeidsgiver AS</BodyShort>
-          </div>
-          <div className="flex-1">
-            <Label>Org.nr. for underenhet</Label>
-            <BodyShort>123456789</BodyShort>
+            {isLoading ? (
+              <Skeleton variant="text" width="100%" />
+            ) : fantIngenPersoner ? (
+              <BodyShort>
+                Fant ingen personer som du har tilgang til å se arbeidsforholdet
+                til. Dobbeltsjekk fødselsnummer og prøv igjen.
+              </BodyShort>
+            ) : error ? (
+              <BodyShort>Kunne ikke hente data</BodyShort>
+            ) : (
+              <BodyShort>{navnMedStorBokstav(data?.navn)}</BodyShort>
+            )}
           </div>
         </div>
       </Informasjonsseksjon>
+      {data && (
+        <Informasjonsseksjon kilde="Fra Altinn" tittel="Arbeidsgiver">
+          {harFlereEnnEttArbeidsforhold ? (
+            <Select label="Velg arbeidsforhold">
+              {data.arbeidsforhold.map((arbeidsforhold) => (
+                <option
+                  key={arbeidsforhold.underenhetId}
+                  value={arbeidsforhold.underenhetId}
+                >
+                  {arbeidsforhold.navn} ({arbeidsforhold.underenhetId})
+                </option>
+              ))}
+            </Select>
+          ) : harEttArbeidsforhold ? (
+            <div className="flex gap-4">
+              <div className="flex-1">
+                <Label>Virksomhetsnavn</Label>
+                <BodyShort>{data.arbeidsforhold[0].navn}</BodyShort>
+              </div>
+              <div className="flex-1">
+                <Label>Org.nr. for underenhet</Label>
+                <BodyShort>{data.arbeidsforhold[0].underenhetId}</BodyShort>
+              </div>
+            </div>
+          ) : (
+            <Alert variant="warning">
+              <BodyLong>
+                Kunne ikke finne noen arbeidsforhold for {data.navn}. Vent litt,
+                og prøv igjen.
+              </BodyLong>
+            </Alert>
```

Comment:
Dette "skal jo ikke skje", men greit å ha en feilhåndtering for det allikevel, tenker jeg.